blog_project/blog_project/urls.py

```diff
@@ -4,8 +4,14 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('blog/',views.HomePage.as_view(),name='homepage'),
-    path('blog/blogs/',views.BlogList.as_view(),name='blog_list'),
-    path('blog/<int:pk>/',views.BlogDetail.as_view(),name='blog'),
-    path('blog/blogger/<int:pk>/',views.AuthorDetail.as_view(),name='author'),
+    path('blog/', views.HomePage.as_view(), name='homepage'),
+    path('blog/blogs/', views.BlogList.as_view(), name='blog_list'),
+    path('blog/<int:pk>/', views.BlogDetail.as_view(), name='blog'),
+    path('blog/blogger/<int:pk>/', views.AuthorDetail.as_view(), name='author'),
+    path('blog/<int:id>/create', views.AddBlogComment.as_view(), name='add_comment'),
+
+    path('accounts/user_registeration/',
+         views.UserRegistration.as_view(), name='user_register'),
+    path('accounts/user_login/', views.UserLogin.as_view(), name='user_login'),
```

Comment:
Please change the endpoint to `accounts/login`. Same for registration, also there is a spelling mistake in registration endpoint.